### PR TITLE
fix doc to be mdx complaint

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
Fixes so docs are mdx complaint

Related content-docs PR: https://github.com/demisto/content-docs/pull/33

## Description
Branch demonstrating how content-docs will generate reference docs based upon a specific branch.

## Docs Fixed
* Gmail (missing images and formatting). 
* AccessData problematic html inside HR

### CODE REVIEW

**Comment by glicht** [comment]
Content-bot comments where a test (deleted). Ignore

File: .github/workflows/docs-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          TEST: ${{ secrets.TEST_SECRET }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+          if [[ "$TEST" == TEST* ]]; then
+            echo "secret is present"
+          else
+            echo "no secret"
+          fi
+      - name: Deploy Docs
+        env:
```
Comment (anara123) [inline .github/workflows/docs-deploy.yml]: I didn't understood, what we exactly this workflow is doing (the use case)?
why we compare ? and why are these `secrets`?

**Comment by glicht** [comment]
Workflow stuff is under test

**Comment by Itay4** [comment]
@glicht can we merge this?

**Comment by glicht** [comment]
It is still work in progress. I use this branch for generating docs.

**Pull request closed without merging**